tigerpath/templates/tigerpath/privacy.html

```diff
@@ -35,24 +35,30 @@ <h1 class="mb-3">Privacy Policy</h1>
       </div>
     </header>
 
-    <section class="text-container">
-      <div class="container">
-        <div>
-          <p>This privacy policy outlines what data we collect about you (the user), where the course offerings and requirements data in TigerPath comes from, and what other third-party services we are using.</p>
-          <h6>Authentication</h6>
-          <p>We protect the entire application underneath CAS authentication, which is the same system you use to sign into other Princeton services, like TigerHub.</p>
-          <h6>User Data</h6>
-          <p>We are committed to protecting your privacy. Your data is stored securely, and we will never sell your data to third parties. We only store the data that we need to in order to provide you with the best experience possible in TigerPath. This data includes your nickname, class year, major, and four-year course schedule. By using TigerPath, you are consenting to the storage of this data.</p>
-          <h6>University Data</h6>
-          <p>We collect the course listings data from the OIT Web Feed Registry and the Course Offerings webpage, and we compile the requirements information from the Princeton University Undergraduate Announcement website and the official Princeton University department websites.</p>
-          <h6>Third-Party Services</h6>
-          <p>We are using some third-party services in order to provide you with the best experience possible:<p>
-          <ul>
-            <li>TigerBook API, to automatically populate your name, class year, and major.</li>
-            <li>TigerApps Transcript API, to help you prepopulate your course schedule with courses you've already taken.</li>
-            <li>Google Analytics, to collect aggregated anonymous metadata on how you use the app so we can continually improve it.</li>
-          </ul>
-        </div>
+    <section class="container mb-4 mt-4">
+      <div class="text-section">
+        <p>This privacy policy outlines what data we collect about you (the user), where the course offerings and requirements data in TigerPath comes from, and what other third-party services we are using.</p>
```

Comment:
_TigerPath aims to allow you to plan your path through Princeton with confidence and ease. Is is our goal to help you be as comfortable as possible when using the app, and so we hold your privacy rights in high regard._

_This privacy policy aims to help you understand what kinds of data we collect about you (the user), where we get the data about course offerings and requirements, and what third-party services we use._

Reply:
The first paragraph is kind of verbose and it seems weird for our goal to be to make the user "comfortable" while using our app

Also, "aims" is repeated in both paragraphs